test/hack/cleanup/cleanup.sh

```diff
@@ -13,10 +13,12 @@ aws cloudformation list-stacks --stack-status-filter CREATE_COMPLETE ROLLBACK_FA
 
 
 # Clean up old launch templates
-launch_templates=$(aws ec2 describe-launch-templates |  jq -r '.LaunchTemplates[].LaunchTemplateName' | grep Karpenter-karpenter-tests)
-n_lts=$(echo ${launch_templates} | wc -w)
+launch_templates=$(aws ec2 describe-launch-templates |  jq -r '.LaunchTemplates[].LaunchTemplateName' | grep "karpenter.k8s.aws")
+n_lts=$(echo "${launch_templates}" | wc -w)
 echo "Removing ${n_lts} launch templates"
-for lt in ${launch_templates}; do
+
+# Iterate through every line in the launch template describe response
+while IFS= read -r lt; do
   echo "Deleting LT ${lt}"
-  aws ec2 delete-launch-template --no-cli-pager --launch-template-name ${lt}
-done
\ No newline at end of file
+  aws ec2 delete-launch-template --no-cli-pager --launch-template-name "${lt}"
+done <<< "$launch_templates"
```

Comment:
changing it like this doesn't handle the case where there are 0 LTs to delete and causes the script to exit 1. 

```suggestion
# Iterate through every line in the launch template describe response
 for lt in ${launch_templates}; do
   echo "Deleting LT ${lt}"
   aws ec2 delete-launch-template --no-cli-pager --launch-template-name "${lt}"
 done
```